.test-defs/lifecycle.yaml

```
@@ -18,4 +18,4 @@ spec:
     --shoot-name=$SHOOT_NAME
     --project-namespace=$PROJECT_NAMESPACE
     --kubecfg="$TM_KUBECONFIG_PATH/gardener.config"
-  image: golang:1.17.5
+  image: golang:1.17.7
```

Any objections against the latest golang version?
```suggestion
  image: golang:1.17.8
```

---

Nope